modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -111,11 +111,6 @@ ComputeMultipleInelasticStress::initialSetup()
 
   _is_elasticity_tensor_guaranteed_constant_in_time =
```

Comment:
It doesn't like you're using `_is_elasticity_tensor_guaranteed_constant_in_time` anymore in this class.  Can you get rid of it?

Reply:
Along those lines, does this PR eliminate all usages of CONSTANT_IN_TIME? If so, you should remove it as well.

Reply:
Re eliminating all constant_in_time stuff - yes, maybe.  I'll change the PR and see what happens.  I suspect that dependent apps might use the const_in_time - we'll see.